.github/workflows/ci.yml

```
@@ -48,6 +48,8 @@ jobs:
           node --run build:types
       - name: Build website
         run: node --run build:website
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
```

https://playwright.dev/docs/browsers
Should we specify to speed things up?
Is `--with-deps` necessary?
```suggestion
        run: npx playwright install chromium --with-deps
```

---

We can try. I followed the CI guide. I can try removing `--with-deps`
https://playwright.dev/docs/ci-intro#on-pushpull_request

---

The time reduced from ~50 seconds to ~6 seconds

---

Nice!